From 1786f0f5c1f3f99e42256f1abd547017073f6566 Mon Sep 17 00:00:00 2001
From: Karthikeyan K <karthikk@habana.ai>
Date: Tue, 08 Mar 2022 10:52:58 +0200
Subject: [PATCH] [SW-77829] boot_desc : update read api

Pub key size, signature size and security version have been removed
from image descriptor structure. It has been moved to the boot desc
header. This patch updates the read api accordingly.

This is in preparation to move the fields into header.

Change-Id: If43b2067cd06e75dadd4315a3996efbde068ef0d
---

diff --git a/common/boot_desc.c b/common/boot_desc.c
index c4bbf70..15aa11b 100644
--- a/common/boot_desc.c
+++ b/common/boot_desc.c
@@ -148,9 +148,6 @@
 			p_img_desc[i].load_high[idx] << 32) |
 			p_img_desc[i].load_low[idx];
 		desc->desc_ver = p_img_desc[i].desc_ver;
-		desc->sec_ver = p_img_desc[i].sec_ver[idx];
-		desc->pub_key_size = p_img_desc[i].pub_key_size[idx];
-		desc->sig_size = p_img_desc[i].sig_size[idx];
 
 		if (img != HL_FW_BOOT_PTR_SPI)
 			memcpy(desc->chksum, p_img_desc[i].chksum2[idx],
